internal/cli/list_opts.go

```diff
@@ -22,11 +22,13 @@ import (
 const (
 	DefaultPage      = 1
 	DefaultPageLimit = 100
+	OmitCountDefault = false
```

Comment:
this is the default for boolean so can be removed

Reply:
updated